docs/reference/templates.rst

```diff
@@ -379,13 +382,14 @@ repository. Each of them requires an entry with the following parameters:
   ``number`` accepts numeric inputs like integers and floats.
   ``enum`` is a special type that restricts the accepted values to those
   set in the ``enum:`` list of values.
-* ``allow_template_update``: When set to ``true``, indicates that this
-  template supports being updated. When the template gets updated, projects
+* ``allow_template_update``: This field is deprecated and will be ignored.
+  It was used to indicates that the template supports being updated. All
+  templates are updatable now. When the template gets updated, projects
   created from it will get updated with the new template files. Defaults to
   ``false``. Also see ``immutable_template_files``.
 * ``immutable_template_files``: A list of file paths inside the template
-  (relative to the project root) that should not be changed by users for
-  ``allow_template_update`` to work. Users changing any of these files will
+  (relative to the project root) that should not be changed by users.
+  Users changing any of these files will
```

Comment:
I think this warning happens only if you have the cli's commit hooks installed right? Perhaps we should clarify this.